assets/js/index.js

```diff
@@ -0,0 +1,366 @@
+// Global declaration of variables
+
+const recentSearchesContainer = $("#recent-searches-container");
+const weatherInfoContainer = $("#weather-info-container");
+const searchForm = $("#search-form");
+
+// reususable modular function to read data from Local Storage
+const readFromLocalStorage = (key, defaultValue) => {
```

Comment:
Your formatting seems to have excessive tabbing on - there's technically nothing wrong with it, but usually we would see a tab size of 2 (it looks like yours is 4)